backend/src/workspace-documents/workspace-documents.controller.ts

```diff
@@ -36,6 +38,36 @@ import { FindWorkspaceDocumentResponse } from "./types/find-workspace-document-r
 export class WorkspaceDocumentsController {
 	constructor(private workspaceDocumentsService: WorkspaceDocumentsService) {}
 
+	@Put(":document_id")
```

Comment:
It seems that there's no @ApiParam decorator.
With this decorator, we can add descriptions for path parameters.

Reply:
Are these descriptions sufficent?

"ID of workspace"
"ID of document to change title"

Reply:
Yes, I think it's sufficient.